examples/simple_system/ibex_simple_system.core

```diff
@@ -36,10 +36,10 @@ parameters:
     description: "Enable the E ISA extension (reduced register set) [0/1]"
 
   RV32B:
-    datatype: int
-    paramtype: vlogparam
-    default: 0
-    description: "Enable the B ISA extension (bit manipulation EXPERIMENTAL) [0/1]"
+    datatype: str
```

Comment:
Isn't it an `int`?

Reply:
Yes it is. But fusesoc does not like enumerated parameters.
We can either put it as an int and pass directly the enumerated value (0 for none, 1 for balanced or 2 for full), or pass it with vlogdefinee as a string and use a macro in ibex_simple_system.sv

Reply:
Ah yes, you are right. I mixed this up. This here is the parameter type for FuseSoc.